features/bevragen/persoon/verblijfplaats/dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,268 @@
+      | gemeente_code                              | 0599                               |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000267                       |
+      | fields              | verblijfplaats.verblijfadres<veld>  |
+#      | fields              | verblijfplaatsBinnenland<veld>  |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+      | naam                             | waarde                             |
+      | straat                           | <straat>                           |
+      | huisnummer                       | <huisnummer>                       |
+      | postcode                         | <postcode>                         |
+      | woonplaats                       | <woonplaats>                       |
+      | locatiebeschrijving              | <locatiebeschrijving>              |
+      | adresseerbaarObjectIdentificatie | <adresseerbaarObjectIdentificatie> |
+      | nummeraanduidingIdentificatie    | <nummeraanduidingIdentificatie>    |
+	  | naamOpenbareRuimte               | <straat>                           |
+
+      Voorbeelden:
+      | veld                 | straat      | huisnummer | postcode | woonplaats | adresseerbaarObjectIdentificatie | nummeraanduidingIdentificatie | locatiebeschrijving          |
+      | .straat              | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
+      | .locatiebeschrijving | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
+      |                      | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
+      | .straat              |             |            |          |            |                                  |                               | Woonboot in de Grotere Sloot |
+      | .locatiebeschrijving |             |            |          |            |                                  |                               | Woonboot in de Grotere Sloot |
+      |                      |             |            |          |            |                                  |                               | Woonboot in de Grotere Sloot |
+
+  Rule: wanneer het veld 'land' de waarde '0000' heeft wordt in plaats daarvan dat veld geleverd als een veld van verblijfplaats m.b.v. de alias 'verblijfplaatsBinnenland' wordt gevraagd zodat het type 'VerblijfplaatsOnbekend' kan worden bepaald.
+
+    Scenario: met fields wordt gevraagd naar een adresveld bij een onbekend adres m.b.v. de alias verblijfplaatsBinnenland
```

Comment:
Volgens mij is het aanleveren van alleen land code en omschrijving niet voldoende. Er kunnen ook andere velden gevuld zijn zoals datum aanvang adreshouding/adres buitenland